prime-dotnet-webapi/Controllers/BannersController.cs

```diff
@@ -24,111 +25,163 @@ public BannersController(IBannerService bannerService)
             _bannerService = bannerService;
         }
 
-        // PUT: api/banners/enrolment-landing
+        // POST: api/banners?locationCode=1
         /// <summary>
-        /// Set the Enrollee landing banner
+        /// Create a banner
         /// </summary>
         /// <param name="viewModel"></param>
-        [HttpPut("enrolment-landing", Name = nameof(SetEnrolleeLandingBanner))]
+        /// <param name="locationCode"></param>
+        [HttpPost("", Name = nameof(CreateBanner))]
         [Authorize(Roles = Roles.PrimeSuperAdmin)]
         [ProducesResponseType(StatusCodes.Status401Unauthorized)]
         [ProducesResponseType(StatusCodes.Status403Forbidden)]
         [ProducesResponseType(typeof(ApiResultResponse<BannerViewModel>), StatusCodes.Status200OK)]
-        public async Task<ActionResult> SetEnrolleeLandingBanner(BannerViewModel viewModel)
+        public async Task<ActionResult> CreateBanner(BannerViewModel viewModel, [FromQuery] BannerLocationCode locationCode)
```

Comment:
Do we still use this endpoint when we also have the more specific ones?

Reply:
Other way around, need this one not the other ones.